pages/demo/html/video.tsx

```diff
@@ -5,6 +5,7 @@ import Header from '~/components/Header';
 import Footer from '~/components/Footer';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.

Reply:
I have just only replace the `<FormattedText>` wherever required.. but i think this extra changes took place due to prettier formatting .... how should i set my settings so as to keep in accordance with u

Reply:
Yep, this is what I think too. Probably when you saved the files, it auto-formatted.

Do you have ESLint extension installed? Maybe this matters as well?

Reply:
Ok let me try ...

Reply:
<img width="381" alt="eslint" src="https://user-images.githubusercontent.com/69234404/125434694-878e5e20-4244-4337-b05c-dc64ffc350e4.PNG">

Are these customizations ok ? or do i have to add something?

Reply:
These are all my settings:
```
{
    "terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
    "[typescript]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "editor.fontFamily": "Fira Code",
    "editor.fontLigatures": true,
    "[javascript]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "[html]": {
        "editor.defaultFormatter": "vscode.html-language-features"
    },
    "[json]": {
        "editor.defaultFormatter": "vscode.json-language-features"
    },
    "workbench.colorTheme": "Dracula",
    "[vue]": {
        "editor.defaultFormatter": "octref.vetur"
    },
    "[javascriptreact]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[scss]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[typescriptreact]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "[jsonc]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[css]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "prettier.singleQuote": true,
    "[xml]": {
        "editor.defaultFormatter": "redhat.vscode-xml"
    },
    "workbench.editorAssociations": {
        "git-rebase-todo": "default"
    },
    "window.zoomLevel": 2,
}
```

Reply:
specifically do i need to ignore `Dangerous property 'dangerouslySetInnerHTML' found eslintreact/no-danger` and `eslint/noused-vars` and how do i ignore them ?

Reply:
I think you just need to isntall the eslint extension if you're running VSCode: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

Reply:
And these are my Workspace settings:

```
{
  "eslint.format.enable": true,
  "eslint.validate": [
    "javascript",
    "javascriptreact",
    "typescript",
    "typescriptreact"
  ],
  "editor.tabSize": 2,
}

```

Reply:
![image](https://user-images.githubusercontent.com/11327761/125435766-73ef9cad-a112-4a37-b653-dce65269be15.png)